tox.ini

```diff
@@ -0,0 +1,19 @@
+[flake8]
+max-line-length = 88
+
+[tox]
+skipsdist = True
+envlist = lint
+
+[testenv]
+basepython = python3
```

Comment:
Is this line actually necessary? I haven't run into any issues in a while where `python` was pointing at a `python2` binary.

Reply:
lets find out!